test/count.js

```diff
@@ -0,0 +1,26 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+describe('count', function() {
+  var even = function(x) { return x % 2 === 0; };
+  it('counts items in a list that match a filter', function() {
+    eq(R.count(even, [1, 2, 3, 4, 5]), 2);
+    eq(R.count(even, [1, 2, 3, 6]), 2);
+    eq(R.count(even, [1, 2, 3]), 1);
+  });
+
+  it('returns zero when no element matches', function() {
+    eq(R.count(even, [1, 3, 5, 7]), 0);
+  });
+
+  it('returns zero for an empty list', function() {
+    eq(R.count(even, []), 0);
+  });
+
+  it('is curried', function() {
```

Comment:
While some old function still have this test, we generally skip it for new function.  That's just how the library works.  You can work curried calls into some of the earlier examples without calling it out directly.

Reply:
I'm more on a side that having tests is better than not having tests.

But if it's conventional here I'm fine with removing 👍

Reply:
DONE